test_room.py

```diff
@@ -0,0 +1,49 @@
+from user import User
+from room import room
+import unittest
+import random
+random.seed(50)
+
+
+class RoomTest(unittest.TestCase):
+    user1 = User("harshit")
+    user2 = User("mukul")
+    user3 = User("ankit")
+    user4 = User("mayank")
+    room1 = room(user1, user2, user3, user4)
+
+    def test_method1(self):
```

Comment:
This looks more like an integration-test rather than a unit-test. Unit tests should focus on testing the functionality of a single unit - like a function - correctly.